util/admin/admin.go

```diff
@@ -127,7 +128,9 @@ func CancelJobs(txn kv.Transaction, ids []int64) ([]error, error) {
 		found := false
 		for j, job := range jobs {
 			if id != job.ID {
-				log.Debugf("the job ID %d that needs to be canceled isn't equal to current job ID %d", id, job.ID)
+				log.Debug("the job that needs to be canceled isn't equal to current job",
```

Comment:
please use logutil

Reply:
Using a background context is not helpful either.

Reply:
according to the discussing with @zimulala last few days ago, we should use logutil even if there is no valid context 😂

Reply:
Yes, we discussed it on engine channel, except for the people in their group, it seems that only @lysu and I are on that channel.
Sorry, I didn't inform you.